### PR TITLE
update version numbers

### PR DESCRIPTION
**Pull request by doggystylez**
## What is the purpose of the change

updates chain.schema.json

## Brief Changelog

update osmosisd and cosmos sdk version numbers, binary links

added "v" in `recommended_version` number to unify with chain registry

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (no)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (no)
  - How is the feature or change documented? (not applicable)

### CODE REVIEW

File: chain.schema.json [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+      "v12.2.0"
     ],
     "binaries": {
-      "linux/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v12.1.0/osmosisd-12.1.0-linux-amd64?checksum=sha256:44433f93946338b8cb167d9030ebbcfe924294d95d745026ada5dbe8f50d5010",
-      "linux/arm64": "https://github.com/osmosis-labs/osmosis/releases/download/v12.1.0/osmosisd-12.1.0-linux-arm64?checksum=sha256:ef2c3d60156be5481534ecb33f9d94d73afa38a1b016e7e1c6d3fe10e3e69b3a"
+      "linux/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v12.2.0/osmosisd-12.2.0-linux-amd64?checksum=sha256:b4a0296b142b1a535f3116021d39660868a83fc66b290ab0891b06211f86fd31",
+      "linux/arm64": "https://github.com/osmosis-labs/osmosis/releases/download/v12.2.0/osmosisd-12.2.0-linux-arm64?checksum=sha256:84717e741b61ef3616fef403aae42067614e58a0208347cd642f6d03240b7778"
     },
-    "cosmos_sdk_version": "0.45",
+    "cosmos_sdk_version": "0.46",
```
Comment (ValarDragon) [inline chain.schema.json:13]:
```suggestion
    "cosmos_sdk_version": "0.45",
```

despite the go.mod, this is actually v0.45. Go mod tooling for minor tracking doesn't play well with replace directives

**Review by ValarDragon — APPROVED** [review]
LGTM, besides on edit, ty!

File: chain.schema.json (hunk 1 above)
Comment (doggystylez) [reply chain.schema.json:13]: TY for that!

**Review by alexanderbez — APPROVED** [review]
🌮

**Pull request merged**